modules/openapi-generator/src/main/resources/typescript-node/model.mustache

```diff
@@ -65,5 +66,15 @@ export namespace {{classname}} {
 {{/vars}}
 }
 {{/hasEnums}}
+{{/isEnum}}
+{{#isEnum}}
+export enum {{classname}} {
+    {{#allowableValues}}
+    {{#enumVars}}
+    {{name}} = <any> {{{value}}}{{^-last}},{{/-last}}
```

Comment:
is there any particular reason for using `<any>` here?
it seems to work out without it e.g. here
https://github.com/OpenAPITools/openapi-generator/blob/b797662eaf8f5a06cbeda1df9d53bc0ce06f8429/modules/openapi-generator/src/main/resources/typescript-fetch/modelEnum.mustache#L6-L12

Reply:
I just copied this part from the already existing code for enums inside a request. "<any>" was already used in the first typescript node template from Geoff Brown.
Being not a typescript expert myself I just didn't want to change too much.
And I just asked my typescript college, he thinks <any> is not needed...